### PR TITLE
python310Packages.tinycss2: 1.1.0 -> 1.1.1

### PR DESCRIPTION
**Pull request by onny**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/tinycss2/default.nix [hunk 1]
```diff
@@ -4,38 +4,41 @@
...
 , webencodings
 , pytestCheckHook
+, flit-core
 }:
 
 buildPythonPackage rec {
   pname = "tinycss2";
-  version = "1.1.0";
+  version = "1.1.1";
   disabled = pythonOlder "3.5";
```
Comment (fabaff) [inline pkgs/development/python-modules/tinycss2/default.nix:13]: 3.5 is no longer [supported](https://github.com/Kozea/tinycss2/blob/v1.1.1/pyproject.toml#L11).

```suggestion
  disabled = pythonOlder "3.6";
```

**Comment by onny** [comment]
Okay it's fixed @fabaff :)

File: pkgs/development/python-modules/tinycss2/default.nix [hunk 2]
```diff
@@ -4,38 +4,41 @@
...
-
   postPatch = ''
     substituteInPlace pyproject.toml \
       --replace "'pytest-cov', 'pytest-flake8', 'pytest-isort', 'coverage[toml]'" "" \
-      --replace "--isort --flake8 --cov" ""
+      --replace "--isort --flake8 --cov --no-cov-on-fail" ""
   '';
 
+  propagatedBuildInputs = [
+    flit-core
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/tinycss2/default.nix:32]: This should go to nativeBuildInputs

File: pkgs/development/python-modules/tinycss2/default.nix [hunk 3]
```diff
@@ -4,38 +4,41 @@
...
 }:
 
 buildPythonPackage rec {
   pname = "tinycss2";
-  version = "1.1.0";
-  disabled = pythonOlder "3.5";
-  format = "flit";
+  version = "1.1.1";
+  disabled = pythonOlder "3.6";
+  format = "pyproject";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/tinycss2/default.nix:14]:
```suggestion
  format = "flit";
```

https://github.com/Kozea/tinycss2/blob/v1.1.1/pyproject.toml#L2-L3

File: pkgs/development/python-modules/tinycss2/default.nix [hunk 4]
```diff
@@ -4,38 +4,41 @@
...
+    webencodings
+  ];
+
+  checkInputs = [ pytestCheckHook ];
+
   meta = with lib; {
     description = "Low-level CSS parser for Python";
     homepage = "https://github.com/Kozea/tinycss2";
     license = licenses.bsd3;
+    maintainers = with maintaners; [ onny ];
```
Comment (mweinelt) [inline pkgs/development/python-modules/tinycss2/default.nix:42]: typo breaks eval

**Comment by onny** [comment]
Thank you both @SuperSandro2000 and @mweinelt !

**Pull request merged**